tests/integration/pilot/headers_test.go

```diff
@@ -39,6 +39,7 @@ import (
 func TestProxyHeaders(t *testing.T) {
 	framework.NewTest(t).
 		Features("traffic.routing").
+		RequireIstioVersion("1.19").
```

Comment:
1.18 is EOL, do we still need it?

Reply:
The tests are still failing in our side as we have have the clusters with different Istio versions.

Also, it is good to have this config in tests. Adding it now will make it obvious to update this minVersion when changes (or supporting new feature) are made in tests.

Reply:
As mentioned earlier, we have multiple control planes in different versions for Multiversion Stability tests. 
We will be skipping the test if the required istio version is less than the one mentioned in this config. 

It is highly recommended to use RequireIstioVersion for every test which requires a newer Istio version.

Reply:
Our internal product still supports 1.18 release. As Iyswareya said we have multiversion tests where different versions of control planes are installed. We run latest version integration tests for all versions to check that if control planes are upgraded then tests should not break.
Having separate matched integration test according to control planes will not be able to test upgrade path.